index.js

```diff
@@ -151,7 +151,10 @@ handlers[protos.GC.CSGO.Internal.EGCBaseClientMsg.k_EMsgGCClientConnectionStatus
   var status = protos.GC.CSGO.Internal.CMsgConnectionStatus.decode(message).status;
 
   switch (status) {
-    case CSGO.GCConnectionStatus.GCConnectionStatus_HAVE_SESSION:
+    // TODO: This is a temporary fix since CSGO.GCConnectionStatus is undefined. 
```

Comment:
Yeah, looks like `protos.GC.CSGO.Internal.GCConnectionStatus` would be the correct spot to pull `GCConnectionStatus_HAVE_SESSION` from.